packages/components/src/tag/src/TagGroup.tsx

```diff
@@ -128,7 +160,9 @@ function TagGroup(props: TagGroupProps, ref: ForwardedRef<HTMLDivElement>) {
  *
  * [View Documentation](TODO)
  */
-const _TagGroup = forwardRef<HTMLDivElement, TagGroupProps>(TagGroup);
-_TagGroup.displayName = "TagGroup";
+const _TagGroup = forwardRef(TagGroup) as <T>(
```

Comment:
what is happening here?

Reply:
we do this on all the collections or components that have <T> although maybe we can adopt the way spectrum does it (didn't have this before).